src/parser.rs

```diff
@@ -141,28 +137,26 @@ pub fn inline<T>(result: Result<T, ParserError>) -> Result<T, (ParserError, Toke
 	result.map_err(|error| (error, Token::Terminator))
 }
 
-pub fn animation(lexer: &mut Lexer, check_with: bool) -> Result<Option<AnimationDeclaration>, (ParserError, Token)> {
-	if check_with {
-		match inline(lexer.token())? {
-			None | Some(Token::Terminator) => return Ok(None),
-			Some(Token::Identifier(string)) if string == "with" => {}
-			Some(token) => return Err((ParserError::UnexpectedToken, token)),
-		}
+pub fn animation(lexer: &mut Lexer) -> Result<Option<AnimationDeclaration>, (ParserError, Token)> {
+	match inline(lexer.token())? {
+		None | Some(Token::Terminator) => return Ok(None),
+		Some(Token::Identifier(identifier)) if identifier == "with" => (),
+		Some(_) => return Err((ParserError::UnexpectedToken, Token::Terminator)),
 	}
```

Comment:
Add a newline after this match expression

Reply:
Done.